google-login-plugin/build.gradle

```diff
@@ -1,4 +1,5 @@
 // see https://github.com/JetBrains/gradle-intellij-plugin
+import org.apache.tools.ant.filters.ReplaceTokens
```

Comment:
This seems a very strange dependency to add here. Are we already depending on that? Is there some groovy/pure Java way to do this?